src/configure/mod.rs

```diff
@@ -566,15 +587,11 @@ async fn try_smtp_ports(
 ) -> Result<LoginParam> {
     // Try to infer port from socket security.
     if param.send_port == 0 {
-        if 0 != param.server_flags & DC_LP_SMTP_SOCKET_STARTTLS as i32 {
-            param.send_port = 587;
-        }
-        if 0 != param.server_flags & DC_LP_SMTP_SOCKET_PLAIN as i32 {
-            param.send_port = 25;
-        }
-        if 0 != param.server_flags & DC_LP_SMTP_SOCKET_SSL as i32 {
-            param.send_port = 465;
-        }
+        param.send_port = match param.send_security {
+            Socket::Automatic => 0,
+            Socket::STARTTLS | Socket::Plain => 587,
```

Comment:
I have changed heuristic to use port 587 instead of port 25 for plain.

Port 25 is for MTA-MTA transmission, it shoud not be used by Delta Chat at all.